src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -571,6 +571,13 @@ class MessageHelper {
    */
   static bool SendUnsubscribedWayPoints(ApplicationManager& app_mngr);
 
+  /**
+   * @brief Creates SubscribeWayPointsMessage to be sent to HMI
+   * @return filled smart object with relevant message data
+   */
+  static smart_objects::SmartObjectSPtr CreateSubscribeWayPointsMessageToHMI(
```

Comment:
@mked-luxoft I'd prefer make more abstract function.
We have next function below
```
static smart_objects::SmartObjectSPtr CreateMessageForHMI(
hmi_apis::messageType::eType message_type, const uint32_t correlation_id);
```
If I we're you I'd make
```
static smart_objects::SmartObjectSPtr CreateMessageForHMI(
hmi_apis::FunctionID::eType function_id, const uint32_t correlation_id);
```

Reply:
@KVGrygoriev  fixed in a6ab85ca0ac52449928ea83373d788e6c7b429c2